.github/ISSUE_TEMPLATE/epic.yml

```diff
@@ -1,104 +1,69 @@
 - type: textarea
-  id: summary
+  id: press-release
   attributes:
-    label: Summary
-    description: TLDR description of the epic. Give a succinct and plain overview of what the epic is about.
-    placeholder: Give a succinct and plain overview of what the epic is about.
+    label: Press release (and FAQ's)
+    description: What would the future feature announcement look like from a customer/ user point of view? Think about questions both Gitpodders, and users/customers might ask about this feature. 
   validations:
-    required: true
+    required: false
 - type: textarea
   id: context
   attributes:
     label: Context
-    description: What thinking led to this? Provide any necessary historical context required to understand this epic.
-    placeholder: Provide any necessary historical context required to understand this epic.
+    description: What thinking led to this issue? Provide any necessary historical context required to understand this epic.
   validations:
-    required: true
+    required: false
 - type: textarea
   id: value
   attributes:
     label: Value
-    description: Why should we do it? How do we know this is a real problem and worth solving?
-    placeholder: Explicitly describe the value to Gitpod and/or our users. I.e. why answer should we undertake this epic?
+    description: Why should we do this work? How do we know this is a real problem and worth solving?
   validations:
-    required: true
+    required: false
 - type: textarea
   id: acceptance-criteria
   attributes:
     label: Acceptance Criteria
     description: What needs to be done before the work is considered complete? The checks which must be complete for this epic to be considered done.
-    placeholder: Defines clearly when the work is complete. Acts as a litmus test for "done" and avoids "done" being ambiguous. Useful for implicit assumptions, e.g. ensuring docs updates are not forgotten.
   validations:
-    required: true
+    required: false
 - type: textarea
   id: measurement
   attributes:
     label: Measurement
-    description: How will we know whether we've been successful / solved the problem? How will you measure the success of the epic? Ideally this metric is one of our key product metrics.
-    placeholder: Important as it's how we track the outcomes (not just output) of the work and prove a change was worth it. Or it should be removed or iterated.
-  validations:
-    required: true
-- type: textarea
-  id: growth-area
-  attributes:
-    label: Growth Area
-    description: Which aspect of Gitpod do we expect improvements in? Acquisition/Onboarding/Exploration/Expansion as defined in [Funnel Proposal](https://www.notion.so/gitpod/Funnel-Proposal-d7d0dba8aced4184b660092a74f8dd3a) (internal)
-    placeholder: Growth is key. This allows us to frame epics from a growth context. Which areas are we expecting this epic to help us with our growth initiatives?
```

Comment:
Whilst I believe to be valuable, I don't think the growth area model is well understood in Gitpod right now, I suggest removing, and consider re-adding in future.